template.tpl

```diff
@@ -0,0 +1,2413 @@
+                "type": "EQUALS"
+              },
+              {
+                "paramName": "uaEcomFunction",
+                "paramValue": "trackEnhancedEcommerceRemoveFromCart",
+                "type": "EQUALS"
+              },
+              {
+                "paramName": "uaEcomFunction",
+                "paramValue": "trackEnhancedEcommerceCheckoutStep",
+                "type": "EQUALS"
+              },
+              {
+                "paramName": "uaEcomFunction",
+                "paramValue": "trackEnhancedEcommercePurchase",
+                "type": "EQUALS"
+              }
+            ]
+          }
+        ],
+        "enablingConditions": []
+      }
+    ],
+    "enablingConditions": [
+      {
+        "paramName": "ecomAPI",
+        "paramValue": "uaEcommerce",
+        "type": "EQUALS"
+      }
+    ]
+  },
+  {
+    "displayName": "Snowplow Tracker and Ecommerce Plugin Settings",
+    "name": "trackerConfiguration",
+    "groupStyle": "ZIPPY_OPEN",
+    "type": "GROUP",
+    "subParams": [
+      {
+        "type": "GROUP",
+        "name": "trackerSettingsGroup",
+        "displayName": "Tracker Settings",
+        "groupStyle": "ZIPPY_OPEN",
+        "subParams": [
+          {
+            "help": "Set to a Google Tag Manager variable of type \"Snowplow v3 Settings\". This is optional - you can configure the tracker either partly or entirely by the overriding settings option below.",
+            "macrosInSelect": true,
+            "selectItems": [
+              {
+                "displayValue": "Select a Snowplow v3 Settings variable",
+                "value": "select"
+              }
+            ],
+            "displayName": "Tracker Settings",
+            "defaultValue": "select",
+            "simpleValueType": true,
+            "name": "trackerConfigurationVariable",
+            "type": "SELECT",
+            "valueValidators": [
+              {
+                "type": "REGEX",
+                "args": [
+                  "^(?!select)$"
+                ],
+                "enablingConditions": [],
+                "errorMessage": "Please choose a GTM variable of type \"Snowplow v3 Settings\"."
+              }
+            ]
+          },
+          {
+            "type": "GROUP",
+            "name": "advancedTrackerSettingsGroup",
+            "displayName": "Advanced Tracker Settings",
+            "groupStyle": "ZIPPY_CLOSED",
+            "subParams": [
+              {
+                "type": "TEXT",
+                "name": "globalName",
+                "displayName": "Global name",
+                "simpleValueType": true,
+                "defaultValue": "snowplow",
+                "help": "The global function name to be created. \u003cstrong\u003eWarning:\u003c/strong\u003e Changing the default value requires a corresponding change to the template\u0027s \u003cstrong\u003eAccesses global variables\u003c/strong\u003e permission.",
```

Comment:
I know it's mostly for consistency with the other templates, but do we think this is worth offering long-term?
Updating the permissions disconnects you from the Gallery version anyway, I feel like at that point we may as well just make them change a constant in the code. :shrug:

It's also the kind of thing that really should live in the Settings imo, a Tag shouldn't care about it.